src/Blazored.Modal/Configuration/ModalAnimationType.cs

```diff
@@ -3,5 +3,7 @@
 public enum ModalAnimationType
 {
     FadeInOut,
+    PopInOut,
+    PopIn,
     None
-}
\ No newline at end of file
+}
```

Comment:
I'd try to prevent committing these kind of white space changes as not to pollute the `git blame`